python/ray/autoscaler/updater.py

```diff
@@ -54,6 +53,10 @@ def __init__(self,
                  runtime_hash,
                  process_runner=subprocess,
                  use_internal_ip=False):
+
+        ssh_control_path = '/tmp/{}_ray_ssh_sockets/{}'.format(
+            os.environ['USER'], cluster_name)
```

Comment:
This might be obvious, but does `os.environ['USER']` always exist?

Reply:
Also, I think these should be formatter with `"` instead of `'`. I guess the linter should warn about these?

Reply:
Changed `'` to `"` and it's now using the potentially more robust `getpass.getuser()` Python stdlib function:

https://docs.python.org/3/library/getpass.html#getpass.getuser

> getpass.getuser()
Return the “login name” of the user.

> This function checks the environment variables LOGNAME, USER, LNAME and USERNAME, in order, and returns the value of the first one which is set to a non-empty string. If none are set, the login name from the password database is returned on systems which support the pwd module, otherwise, an exception is raised.

> In general, this function should be preferred over os.getlogin().

Reply:
`'` -> `"` in the linting can be done as in this (incomplete) branch https://github.com/ray-project/ray/compare/master...robertnishihara:flakequotes